static/js/publisher/release/components/releasesConfirmDetails/releaseRow.js

```diff
@@ -27,7 +27,7 @@ const ReleaseRow = ({ type, revisionInfo, channel, progress, notes }) => (
         <span className="p-release-details-row__progress">
           <ProgressiveBar percentage={100} disabled={true} />
           <span>
-            <span className="p-tooltip--btm-right">
+            <span className="p-tooltip--btm-center">
               <span className="p-help">100% of devices</span>
               <span className="p-tooltip__message">
                 Cannot progressively release to an empty channel
```

Comment:
When testing this, I really thought it was a bug that one of the sliders was stuck at 100%. Only by accident I triggered the tooltip and noticed that it was different from the other tooltips.

If “Releases are delivered to devices…” was static text below the table, rather than a tooltip, this value being the only one with a dotted underline would have been a hint that the difference was intentional.

Even better, this tooltip would be static text as well. It could go in the row’s rightmost cell, which would be empty otherwise.